prow/github/client.go

```diff
@@ -1312,16 +1314,41 @@ func (c *Client) CreateReview(org, repo string, number int, r DraftReview) error
 	return err
 }
 
+func prepareReviewersBody(logins []string, org string) (map[string][]string, error) {
+	body := map[string][]string{}
+	for _, login := range logins {
+		mat := teamRe.FindStringSubmatch(login)
+		if mat == nil {
+			if _, exists := body["reviewers"]; !exists {
+				body["reviewers"] = []string{}
+			}
+			body["reviewers"] = append(body["reviewers"], login)
+		} else if mat[1] == org {
+			if _, exists := body["team_reviewers"]; !exists {
+				body["team_reviewers"] = []string{}
+			}
+			body["team_reviewers"] = append(body["team_reviewers"], mat[2])
+		} else {
+			return nil, fmt.Errorf("team %s is not part of %s org", login, org)
```

Comment:
If any `login` is for a team in a different org (or malformed) we won't return a body containing the other logins. It is probably preferable to make this 'best effort' by still trying to add other logins to the body and collecting errors in a slice (see [`errorutil`](https://github.com/kubernetes/test-infra/blob/master/prow/errorutil/aggregate.go)).
`tryRequestReview` could then continue upon error from this function (so long as `len(body)>0`) and return a different error type to distinguish client errors from user input errors.